build.py

```diff
@@ -64,6 +64,17 @@ def GetCommandOutput(command):
         pass
     return None
 
+def GetGitHeadInfo(context):
+    """write out HEAD commitID and Date."""
+    try:
+        with CurrentWorkingDirectory(context.mayaUsdSrcDir):
+            commitSha = subprocess.check_output('git rev-parse HEAD').decode()
+            commitDate = subprocess.check_output('git log -1 --pretty=format:"%ad"'.split()).decode()
```

Comment:
We just grabbed the commit in the previous command, so maybe slightly cleaner to get the date this way?:
```
git show <commitSha> --pretty=format:"%ad"
```
Either works though. :)

Reply:
Thanks @mattyjams  good suggestion.  Just one minor thing, `git show <commitSha>` shows diff information which we don't want. passing `-s` did the trick.

Also instead of passing commitSHA, I simply passed HEAD.  Either of these cosmetic works.

`git show -s HEAD --format="%ad"`